### ISSUE TITLE
Accessibility Testing for Design System & Forms library > Form 21-4140 Employment questionnaire

### ISSUE THREAD

**Issue by tbaker1026** [department-of-veterans-affairs/va.gov-team#120468]
[URL For testing](https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/)

### Product information

- [x] Team name, product name, and feature name have been added to the title of this issue.
- [x] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

@jeana-adhoc 

### Use of color and color contrast checks

- [X] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [X] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [X] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [X] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [X] Any charts, maps, infographics, and tables convey all information without only relying on color
- [X] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")
- [ ] (Mobile app only) Color contrast ratios are met for all supported dark, contrast, and inverted color modes
- [ ] (Mobile app only) All elements retain their meaning for all supported dark, contrast, and inverted color modes

### How did color testing go?

Only known issue:
-  https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4683
  - This work is planned in a later epic:
    - https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3750)

### Who completed the axe scans?

@jeana-adhoc 

### axe checks

- [X] Each page or screen has been scanned using axe (results shared below)
- [X] axe is integrated into your end-to-end testing

### axe DevTools scan results

| Axe Results   | Page           |
| ------------- | -------------- |
| Content not in a landmark (true for all pages of this form) | [Introduction](https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140) |
| Same as intro | [Your name and date of birth](https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/name-and-date-of-birth) |
| Same as intro | [Your identification information](https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/identification-information) |
| Same as intro | [Your mailing address](https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/address) |
| Same as intro | [Your phone and email address](https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/phone-and-email-address) |
| Same as intro | [Were you employed the last 12 months?](https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/employers-summary) |
| Same as intro | [Employers info](https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/employers/0/name-and-address?add=true) |
| Same as intro | [Dates employed at $job](https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/employers/0/employment-dates?add=true) |
| Same as intro | [Employment details for $job](https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/employers/0/employment-details?add=true) |
| Same as intro | [Employer summary page](https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/employers-summary) |
| No h1 on the page | [Unemployment](https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/unemployed) |
| Same as intro | [Evidence Upload](https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/evidence) | 
| Same as intro | [Review page](https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/review-and-submit) |
| Same as intro | [Confirmation page](https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/confirmation) |

### How did axe testing go?

Axe revealed a few issues that were already noted in other findings: 

1. The "Skip to content" link is being marked as not being in a landmark area. I think this is related to the minimal form layout. We see the same issue in the [mock minimal form layout](https://staging.va.gov/mock-form-minimal-header/introduction) (Nav to name & dob page) but not in the default/current [mock form layout](https://staging.va.gov/mock-form-patterns/introduction) (Nav to name & dob page). A ticket was already filed for the forms team to look into this:
  - https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/2037
2. The Unemployment page does not have an `H1` which is consequently causing focus to go to the form stepper/progress indicator, which does not happen on any other page in this form. Focus should go the `h1` (which is currently an `h3`). 
  - https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4931

### Who completed the content zoom and reflow test?

@jeana-adhoc 

### Content zoom and reflow checks

- [X] (Web only) All page elements are readable and usable at 200% zoom
- [X] (Web only) All page elements are readable and usable at 300% zoom
- [X] (Web only) All page elements are readable and usable at 400% zoom
- [ ] (Mobile app only) All page elements are readable and usable when device text is at the maximum possible size

### How did content zoom and reflow testing go?

One issue found.

At 400%, the starting form over modal is not fully within the viewport.  It is not causing a horizontal scroll. This is an issue in all forms, not just this one. The modal is triggered from the intro page (when returning to a form you had previously started or from the form-saved page (after you click "finish this application later).

<img width="400" height="471" alt="Image" src="https://github.com/user-attachments/assets/1a125abc-5bd3-47ec-9c8c-3db438fd26c1" />

A later modal, in the list and loop does fit fully within the viewport

<img width="400" height="451" alt="Image" src="https://github.com/user-attachments/assets/0ee5a377-6cab-49ff-a1b8-940d34a7ec36" />

- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4932

### Who completed the keyboard navigation test?

@jeana-adhoc

### Keyboard navigation checks

- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [  ] All elements under focus have a visible focus indicator
- [X] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

The confirmation page lost visible focus. Focus seems to be placed above the "[Check the status of your form on My VA](https://staging.va.gov/my-va#benefit-applications)" link, as when I click tab, focus goes to that link. I expect focus, and the focus ring to appear on the `H2` "Form submission started on October 2, 2025"

- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4934

### Do you have any other results to share?

**Issue 1 - Introduction page h1 receives focus**
The introduction page `h1` is receiving focus when it should not. Focus should not be managed on a forms intro page regardless of how the user gets to the introduction page.  This was flagged as a staging review finding in #113288, and is not something we should be doing in our forms. (WCAG Success Criteria: [WCAG 2.4.3 Focus Order - Level A](https://www.w3.org/TR/WCAG22/#focus-order))
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4941

**Issue 2 - Breadcrumb truncation** 
Breadcrumb needs `wrapping="true"` applied to it
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4935

**Issue 3 - Date Validation**
Date validation
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4920

**Issue 4 - Confirmation page**
In the accordion, if no file is uploaded , Instead of displaying "Upload your supoorting evidence", It would be better to say "No evidence uploaded."
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4936

**Issue 5 - Form allows Unauth logins**
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4917

**Issue 6 - Content changes**
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4926
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4927
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4937 
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4930
